### DISCUSSION TITLE
1.0 release

### DISCUSSION THREAD

**Discussion by fobos531** [nandorojo/zeego#28] [Q&A] [answered]
Hey, just a simple question, don't want to rush anything:

- do you have an expected timeframe of when you're aiming for the 1.0 release? Would love to migrate my codebase to the all the goodies that 1.0 brings like submenus, extended customizability etc.

**Comment by nandorojo** [comment, marked as answer]
Hey! I’m using this now and it’s working well. Just need to release one fix for Android and it should be good to go.

**Reply by nandorojo** [reply]
v1 is now live! @fobos531
